docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -5,36 +5,45 @@
-Key Infrastructure (PKI) realm]. To do so, {es} needs to verify the signature on the {kib} client certificate, and it also needs to map the
-client certificate's distinguished name (DN) to the appropriate `kibana_system` role.
-
-NOTE: Using a PKI realm is a gold feature. For a comparison of the Elastic license levels, see https://www.elastic.co/subscriptions[the
-subscription page].
-
-To configure {kib} and {es} to use mutual TLS authentication:
-
-. <<using-kibana-with-security,Set up {kib} to work with {stack} {security-features} with a username and password>>.
-
-. <<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>.
-+
-This entails generating a "server certificate" for {es} to use on the HTTP layer.
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption
+for data-in-transit. While these terms are often used interchangeably, {kib}
+supports only TLS, which supersedes the old SSL protocols.
+
+TLS requires X.509 certificates to authenticate the communicating parties and
+perform encryption of data-in-transit. Each certificate contains a public key
+and has and an associated -- but separate -- private key; these keys are used
+for cryptographic operations. {kib} supports certificates and private keys in
+PEM or PKCS#12 format.
+
+In a standard TLS configuration, the server presents a signed certificate to
+authenticate itself to the client. In a mutual TLS configuration, the client
+also presents a signed certificate to authenticate itself to the server.
+
+{es} {security-features} are enabled on your cluster by default, so each request
+that {kib} (the client) makes to {es} (the server) is authenticated. Most
+requests made by end users through {kib} to {es} are authenticated by using the
+credentials of the logged-in user.
+
+If {kib} has `elasticsearch.username` and `elasticsearch.password` configured,
+it attempts to use these credentials to authenticate to {es} via the
+{ref}/native-realm.html[native realm]. However, {kib} also supports mutual TLS
+authentication with {es} via a
+{ref}/pki-realm.html[Public Key Infrastructure (PKI) realm]. With this setup,
+{es} needs to verify the signature on the {kib} client certificate, and it also
+needs to map the client certificate's distinguished name (DN) to the appropriate
+`kibana_system` role.
+
+NOTE: Using a PKI realm is a gold feature. For a comparison of the Elastic
+license levels, see https://www.elastic.co/subscriptions[the subscription page].
+
+[discrete]
+==== Configure {kib} and {es} to use mutual TLS authentication
+
+When you {ref}/configuring-stack-security.html[start the {stack} with security enabled],
+password authentication for {kib} is configured and TLS encryption is enabled
+between {kib} and {es} automatically. For steps to manually set up TLS
+encryption, refer to
+{ref}/security-basic-setup-https.html#encrypt-kibana-elasticsearch[encrypting traffic between {kib} and {es}].
```

Comment:
I think this is a bit confusing. I'd suggest something along these lines:

"If you haven't already done so, start Kibana and connect it to Elasticsearch using the [enrollment token] (link). This will configure Kibana to authenticate to Elasticsearch using a service account token, and Kibana will trust Elasticsearch's server certificate."

Reply:
Given the changes in [this comment](https://github.com/elastic/kibana/pull/118088#discussion_r747571434), I think that we shorten this sentence since we're already explaining the connection between the enrollment token and service account token:

>If you haven't already, start Kibana and connect it to Elasticsearch using the enrollment token.